app/components/channel_drawer/channel_drawer.js

```diff
@@ -381,9 +381,12 @@ export default class ChannelDrawer extends Component {
     };
 
     render() {
-        const {children} = this.props;
+        const {children, isLandscape} = this.props;
         const {openDrawerOffset} = this.state;
 
+        const androidTop = isLandscape ? 46 : 56;
+        const iosTop = isLandscape ? 32 : 64;
```

Comment:
Should these numbers be constants? Looks like they're used in other places. Maybe something like ANDROID_TOP_LANDSCAPE, ANDROID_TOP_PORTRAIT, etc...